topo_processor/cog/create_cog.py

```diff
@@ -1,10 +1,19 @@
 import os
 
 from topo_processor.cog.command import Command
+from topo_processor.file_system.get_fs import is_s3_path
+from topo_processor.util.aws_credentials import get_credentials
 
 
 def create_cog(input_path: str, output_path: str, compression_method: str) -> str:
     cmd = Command("gdal_translate", {"container": "osgeo/gdal", "tag": "ubuntu-small-latest"})
+
+    if is_s3_path(input_path):
+        cmd.env(f"AWS_ACCESS_KEY_ID={get_credentials().access_key}")
```

Comment:
credentials may vary based on source/destination, 

I think we need a `get_credentials(path)` which then splits it into the `bucket` and `key` component and loads the credentials based off the `bucket` name failing back to the OS `AWS_PROFILE` if none are configured.